Getting-Started.adoc

```diff
@@ -39,29 +39,35 @@ helm init --service-account=tiller
 
 NOTE: To verify that the Tiller pod has started execute the following command: `kubectl get pod --namespace kube-system` and you should see the `tiller` pod running.
 
-==== Add the Sk8s repository to your Helm environment
+==== Add the Riff repository to your Helm environment
 
-Run the following commands to add the `sk8s-incubator` repository:
+Run the following commands to add the repository:
 
 ----
-helm repo add sk8s-incubator http://sk8s-incubator.cfapps.io
-helm repo update
+helm repo add riffrepo https://riff_charts.storage.googleapis.com
+helm repo update riffrepo
 ----
 
-==== Install Sk8s FaaS helm chart
+==== Install Riff FaaS helm chart
 
-Install the Sk8s Helm chart using one of the following commands.
+Install the Riff Helm chart as follows:
+
+- `helm search riffrepo/riff` and identify the version you're looking to install (use latest if unsure)
+- Download the starter install script: `curl -sL https://riff_charts.storage.googleapis.com/riff-<version>-install-example.sh > riff_install.sh`
```

Comment:
I think here we should show the instructions for installing via a simple `helm install` with the latest stable release which will match the version in `Chart.yaml` as soon as we kick off the first such release, even if that means kicking off the first release immediately before merging the updated guide (cc @trisberg).

The instructions for downloading a script to install a specific build version via overrides should be either in a separate Developer Guide or at least in a section that follows completion of the "normal" Getting Started experience.

We also need to provide a `values-snapshot.yaml` that overrides all values to: `x.y.z.-snapshot` where `x.y.z` is the next planned release. Those tags would match the images built locally using the various `build-*` scripts within this project. Instructions for deploying the locally built images should also follow the "normal" steps. Personally I think the order should be:

1. full instructions with a simple `helm install` using the (latest stable) version as defined in `helm-charts/Chart.yaml`
2. supplement section that provides an example `helm install` command using `values-snapshot.yaml` to override for locally built image tags
3. supplement section that provides instructions for downloading the script used by a specific build version

thoughts?